src/mscorlib/shared/System/StringComparer.cs

```diff
@@ -170,6 +171,7 @@ public int GetHashCode(object obj)
     }
 
     [Serializable]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     internal sealed class CultureAwareComparer : StringComparer
```

Comment:
Does the serialization on .NET Core still works with this change? These internal types do not exist in mscorlib on .NET Core. I suspect that we may need to add these internal types to the mscorlib façade in corefx somehow.